nautobot/extras/models/change_logging.py

```diff
@@ -28,16 +29,21 @@ class ChangeLoggedModel(models.Model):
     class Meta:
         abstract = True
 
-    def to_objectchange(self, action):
+    def to_objectchange(self, action, related_object=None, object_data_extra=None, object_data_exclude=None):
         """
         Return a new ObjectChange representing a change made to this object. This will typically be called automatically
         by ChangeLoggingMiddleware.
         """
+
+        extra = {} if not related_object else {"related_object": related_object}
```

Comment:
Seems unnecessary to me - why not just `return ObjectChange(...., related_object=related_object)` instead of the layer of `extra` indirection?